sources/lib/jam/tests/jam-test-suite.lid

```diff
@@ -4,7 +4,6 @@ Author:       Peter S. Housel
 Files:        library
               jam-test
               specification
-Target:       dll
```

Comment:
These were added to prevent linking an executable (in bin) by default when building the project; don't delete them.

Reply:
I think the compiler only looks at `Target-type: dll`?

Reply:
(which is also in this LID file)